configs/openSUSE/permissions-whitelist.toml

```diff
@@ -39,13 +39,21 @@ package = "sendmail"
 type = "permissions"
 note = """Be careful about calculating digest sums here, the sendmail package
       replaces file contents during RPM build time with architecture dependent
-      values"""
+      values.
+      The duplicate entries are due to the UsrMerge. We can remove the /etc
+      entries once the dust has settled. See bsc#1219339"""
```

Comment:
Now that we have a bug for this could you also add a `bug = "bsc#..."` field

Reply:
Added